mynearwallet/my-near-wallet
# manual import account

## Issues
a new feature allow user to import account manually, it skips indexer check in case their account not indexed.
it check for access key list that match with public from their seedphrase/private key


## Changes description
add a new modal manual import as option when import with seedphrase / with private key

## Preview
<img width="789" alt="Screen Shot 2024-03-20 at 12 40 33" src="https://github.com/mynearwallet/my-near-wallet/assets/22742847/6553f4ae-b453-43ca-a9de-df79bf693e9d">
<img width="789" alt="Screen Shot 2024-03-20 at 12 40 03" src="https://github.com/mynearwallet/my-near-wallet/assets/22742847/b2f3286f-d657-4ef4-8aef-c4b751638dcb">

## Tested
- import seedphrase success
- manual import seedphrase success
- manual import seedphrase with wrong accountId / wrong seedphrase, will show error message
- import private key success
- manual import private key success
- manual import private key with wrong accountId / wrong private key, will show error message
- import ledger success
- import ledger reject
- import with unmatch access key should show error
